doc/conftest.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Test configuration file."""
+
+from __future__ import print_function, division, absolute_import
+
+import os
+import sys
+
+from odl.trafos.backends import PYFFTW_AVAILABLE, PYWT_AVAILABLE
+from odl.tomo.backends import (
+    ASTRA_AVAILABLE, ASTRA_CUDA_AVAILABLE, SKIMAGE_AVAILABLE)
+
+# We don't bother excluding stuff depending on available backends. We just
+# require them all (except stir)
+if not all([PYFFTW_AVAILABLE, PYWT_AVAILABLE,
+            ASTRA_AVAILABLE, ASTRA_CUDA_AVAILABLE, SKIMAGE_AVAILABLE]):
+    print('to be able to test the docs, you need to install all backends, '
+          'currently: '
+          '`astra`, `pyfftw`, `pywt`, `skimage`; '
+          'please consult the installation documentation for details',
+          file=sys.stderr)
+    sys.exit(-1)
+
+here = os.path.normpath(os.path.dirname(__file__))
+collect_ignore = [os.path.join(here, 'sphinxext')]
+collect_ignore = [os.path.normcase(ignored) for ignored in collect_ignore]
```

Comment:
Dislike overwriting names like this.

Reply:
What's wrong with it? When it's 100 lines apart, okay, but two consecutive lines? Why use a new name?

Reply:
```
collect_ignore = [os.path.normcase(os.path.join(here, 'sphinxext'))]
```

whats wrong with that?

Reply:
Nothing, but how about other files with more entries in this list? Use `os.path.normcase(os.path.join())` for each entry? Seems like a lot of noise to me.